lib/instrumental.js

```diff
@@ -97,9 +101,17 @@ function instrumental_send(payload) {
     // Connetion Established. Lets go shopping
     state = "connected";
 
+    var cleanString = function(value) {
+      return String(value).replace(/\s+/, "_");
```

Comment:
I feel like this should be `#gsub`.

`#replace` is… [different](http://apidock.com/ruby/String/replace).